src/core/isBichain.js

```diff
@@ -0,0 +1,11 @@
+/** @license ISC License (c) copyright 2016 original and current authors */
```

Comment:
Gonna wanna add a file in predicates that references this core file and export it out on `predicates/index`

Reply:
Should be 2019